### ISSUE TITLE
Changing characters from anyone but Sonic in Global and then starting directly from a level scene throws up null instance error

### ISSUE THREAD

**Issue by Candescence** [Techokami/SonicWorldsNext#86]
So this is what I got when trying to directly run a level via play current scene, setting the character in the Global script. Everyone but Sonic triggers the below error, making it impossible do so with any character other than Sonic.

![image](https://github.com/Techokami/SonicWorldsNext/assets/9062079/c9901d94-c14f-4e58-9bcc-642ba75446bc)

**Comment by Renhoex** [comment]
we can look into it updating this in the near future.

If you really need to test a level with a specific character the current method is to open "CharacterSelect.gd" in the character select.
"res://Scene/Presentation/CharacterSelect.tscn"
add a label for your level on line 11
something like `var levelLabels = ["Base Zone Act 1", "Base Zone Act 2", "YOUR LEVEL NAME"]`
and set a match number for the room to load on line 88-94

**Comment by Candescence** [comment]
> we can look into it updating this in the near future.
> 
> If you really need to test a level with a specific character the current method is to open "CharacterSelect.gd" in the character select. "res://Scene/Presentation/CharacterSelect.tscn" add a label for your level on line 11 something like `var levelLabels = ["Base Zone Act 1", "Base Zone Act 2", "YOUR LEVEL NAME"]` and set a match number for the room to load on line 88-94

Yeah, I already did that, it's just slightly more convoluted and takes a bit more time than trying to open a new level directly, but thanks.

**Comment by DimensionWarped** [comment]
@Renhoex Any chance you can add some global toggle to speed up the splash screen and skip over the title screen as part of this effort?

**Comment by Candescence** [comment]
> @Renhoex Any chance you can add some global toggle to speed up the splash screen and skip over the title screen as part of this effort?

This would be legit fantastic, honestly, would probably shave off quite a bit of time.

**Comment by DimensionWarped** [comment]
Yeah, I pretty much figured the main reason you'd need to set the characters up in globals was in an effort to shave off a little time in getting to your level, and I've definitely felt some of the waiting pain there myself.

(nowhere near as much as in my actual job though where deployment can easily take over an hour...)

**Comment by Renhoex** [comment]
> 

Yeah I'll look into it when I can, there's code to check if the scene was loaded from the shortcut button so I'll probably just add a fast launch option

**Comment by Caverns4** [comment]
Fixed as of commit #235
